pkgs/games/gnujump/default.nix

```diff
@@ -14,7 +14,7 @@ stdenv.mkDerivation rec {
   ];
 
   meta = with stdenv.lib; {
-    homepage = "https://jump.gnu.sinusoid.es/";
```

Comment:
`"https://jump.gnu.sinusoid.es/"` works perfectly well

Reply:
It redirects though

Reply:
(I do not have a strong opinion about this, it's just an url. just tell me if I have to remove that commit)

Reply:
Should be fine.